NetJSON.Tests/SerializeTests.cs

```diff
@@ -471,7 +472,7 @@ public void SerializeNonPublicType()
             var date = DateTime.Now;
             var djson = NetJSON.Serialize(date);
             var ddate = NetJSON.Deserialize<DateTime>(djson);
-            Assert.IsTrue(date == ddate);
+            Assert.IsTrue(date == ddate.ToLocalTime());
```

Comment:
I am not sure i am comfortable with explicitly having to call ToLocalTime() for each test method. The goal is to preserve the transformation without additional effort on the user side. Similar to using JSON.Net. Is there a way to avoid this additional calls? Doing the current logic will break lot of existing code.

Reply:
There are other options yet they all require some `kind` alignment as `DateTime` equality comparison does not include `kind`-related logic. As stated on MSDN the values must be in the same timezone before comparison. Thus, in this particular place there is no option but align timezones by calling either `ToLocalTime` or 'ToUniversalTime'. I am pretty sure this kind of alignment is not mandatory everywhere, only in tests involving timezones.

Reply:
Sure thing. Just let me know when you're done so I can pull the updates and think about the further changes in the way you'd be happy to accept.

Reply:
Does AppVeyor actually give you a successful build after your changes? based on what i am seeing so far, it does not like it does unless I need to make some changes on my part to make it possible.

Reply:
https://ci.appveyor.com/project/AlexSeleznyov/netjson/build/1.0.14 this is the link to the project. VS2015 has some issues with .Net Core so all I cared to have working is VS2017